src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-11-1-upgrade-notes.md

```diff
@@ -79,5 +85,95 @@ Removed variables can be safely deleted from your themes or elements. Deprecated
 * Removed `--nuxeo-results-view-min-height` (added in [NXP-27652](https://jira.nuxeo.com/browse/NXP-27652)).
```

Comment:
This one is not necessarily a breaking change from 10.10, because it was added and removed for 11.1. But we can still leave a note, on the deprecations, maybe?

Reply:
Yes, we can leave a note for it 👍